user/tests.py

```diff
@@ -50,6 +61,119 @@ def json(self):
+        User.objects.all().delete()
+
+    def test_signup_success(self):
+        client = Client()
+        User = {
+            'email' : 'soo123@gmail.com',
+            'password' : '12345Aa!',
+            'check_password' : '12345Aa!'
+        }
+        response = client.post('/user/sign-up', json.dumps(User), content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+    def test_signup_userexists(self):
+        client = Client()
+        User = {
+            'email' : 'soohyung0121@gmail.com',
+            'password' : '12345Aa!',
+            'check_password' : '12345Aa!'
+        }
+        response = client.post('/user/sign-up', json.dumps(User), content_type='application/json')
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.json(), {'message' : 'USER_ALREADY_EXISTS'})
+
+    def test_signup_emailerror(self):
+        client = Client()
+        User = {
+            'email' : 'soohyung0121gmail.com',
+            'password' : '12345Aa!',
+            'check_password': '12345Aa!'
+        }
+        response = client.post('/user/sign-up', json.dumps(User), content_type='application/json')
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.json(), {'message' : 'EMAIL_VALIDATION_ERROR'})
+
+    def test_signup_keyerror(self):
+        client = Client()
+        User = {
+            'id' : 'soohyu@gmail.com',
+            'password' : '12345Aa!',
+            'check_password': '12345Aa!'
+        }
+        response = client.post('/user/sign-up', json.dumps(User), content_type='application/json')
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.json(), {'message' : 'INVALID_KEYS'})
+
+    def test_check_password(self):
+        client = Client()
+        User = {
+            'email' : 'soohg0121@gmail.com',
+            'password' : '12345Aa!',
+            'check_password' : '12345aA!'
+        }
+        response = client.post('/user/sign-up', json.dumps(User), content_type='application/json')
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.json(), {'message' : 'PASSWORD_INCONSISTENCE'})
+
+class SignInTest(TestCase):
+    def setUp(self):
+        hashed_password = bcrypt.hashpw('12345aA!'.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
+        User(
+            email = 'soohyung0121@gmail.com',
+            password = hashed_password
+        ).save()
+
+    def tearDown(self):
+        User.objects.all().delete()
+
+    def test_signin(self):
+        client = Client()
+        User = {
+            'email' : 'soohyung0121@gmail.com',
+            'password' : '12345aA!'
+        }
+        response = client.post('/user/sign-in', json.dumps(User), content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+        access_token = jwt.encode({'email': User['email']}, SECRET_KEY, algorithm='HS256').decode('utf-8')
```

Comment:
algorithm 도 SECRET_KEY 처럼 리팩토링할때 바꿔주세요.
```suggestion
        access_token = jwt.encode({'email': User['email']}, SECRET_KEY, algorithm='HS256').decode('utf-8')
```